internal/api/handlers/operations_handler.go

```diff
@@ -64,6 +64,12 @@ func (h *OperationsHandler) ListOperations(ctx context.Context, request *api.Lis
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
 
+	operationStatus := request.Status
+	if err != nil {
+		handlerLogger.Error(fmt.Sprintf("error parsing sorting parameters, orderBy: %+v", request.Status), zap.Error(err))
+		return nil, status.Error(codes.InvalidArgument, err.Error())
+	}
```

Comment:
Why did this error validation is being done after an assignment to `operation.Status`? What do you think to move it close to which it was assigned?